CMakeLists.txt

```diff
@@ -63,6 +63,10 @@ add_definitions(-DOPENRCT2_COMMIT_SHA1_SHORT="${OPENRCT2_COMMIT_SHA1_SHORT}")
 
 INCLUDE(FindPkgConfig)
 
+if (APPLE)
+	set(ENV{PKG_CONFIG_PATH} "$ENV{PKG_CONFIG_PATH}:/usr/local/opt/openssl/lib/pkgconfig")
+endif (APPLE)
```

Comment:
Hmm, OpenSSL has worked just fine for me without this change. Had a bit of trouble getting it recognised by cmake after installing it through brew, but in the end it worked as long as it was brewn universally.